README.md

```diff
@@ -59,7 +59,13 @@ want to start reading from.
 
 ```javascript
 state.length //=> 0;
-state.get(0) //=> null;
+let record = state.get(0);
+record.isRequested //=> false
+record.isPending //=> false
+record.isResolved //=> false
+record.index //=> null
+record.content //=> null
+record.page //=> { offset: null, size: 0, data: [] }
```

Comment:
Proposed values of an Unrequested Page. Should offset be `null` or the actual offset if it did exist?